Stream.java

```diff
@@ -0,0 +1,62 @@
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class Stream {
+
+    public static void main(String[] args) {
+
+        //StreamAPI未使用.ver
+        System.out.println("--StreamAPI未使用--");
+        List<Integer> integerList = Arrays.asList(1, 2, 3, 4, 5);
+        for (Integer i : integerList) {
+            if (i % 2 == 0) {
+                System.out.println(i);
+            }
+        }
+
+        //Stream<>使用.ver
+        System.out.println("--Stream<T>--");
+        System.out.println("--偶数--");
+        integerList = Arrays.asList(6, 7, 8, 9, 10);
+        integerList.stream()
+                .filter(i -> i % 2 == 0)
+                .forEach(System.out::println);
+
+        System.out.println("--昇順--");
+        integerList = Arrays.asList(6, 7, 8, 9, 10);
+        integerList.stream()
+                .sorted()
+                .forEach(System.out::println);
+
+        System.out.println("--降順--");
+        integerList = Arrays.asList(6, 7, 8, 9, 10);
+        integerList.stream()
+                .sorted(Comparator.reverseOrder())
+                .forEach(System.out::println);
+
+        System.out.println("--map使用し２倍で出力--");
+        integerList = Arrays.asList(6, 7, 8, 9, 10);
+        integerList.stream()
+                .map(i -> i * 2)
+                .forEach(System.out::println);
+
+
+        //IntStream使用
+        System.out.println("--IntStream--");
+        var list = new int[]{6, 7, 8, 9, 10};
+        {
+            System.out.println("--合計--");
+            System.out.println(IntStream.of(list).sum());//合計
+            System.out.println("--最大値--");
+            System.out.println(IntStream.of(list).max().orElse(0));//最大値
+            System.out.println("--最小値--");
+            System.out.println(IntStream.of(list).min().orElse(0));//最小値
+            System.out.println("--平均値--");
+            System.out.println(IntStream.of(list).average().orElse(0));//平均値
+        }
```

Comment:
可読性を意識して `{}` で囲ってくださったのでしょうか？
もしそうだとしたら大変ありがたいです！

ただし、このようにコードの途中で`{}`を使うのは変数のスコープを`{}`内に限定したいときにつかうのでレイアウト的な意味合いでは使いません。

今回の対応としては
```suggestion
        System.out.println("--合計--");
        System.out.println(IntStream.of(list).sum());//合計
        System.out.println("--最大値--");
        System.out.println(IntStream.of(list).max().orElse(0));//最大値
        System.out.println("--最小値--");
        System.out.println(IntStream.of(list).min().orElse(0));//最小値
        System.out.println("--平均値--");
        System.out.println(IntStream.of(list).average().orElse(0));//平均値
```
のようにしてもらえればよいです！

スコープ？となっていればぜひ変数のスコープについて調べてみてください。
これについては余裕があればで大丈夫です。

Reply:
無意識に囲ってしまっていたみたいです汗

Reply:
<img width="293" alt="スクリーンショット 2022-09-11 16 05 21" src="https://user-images.githubusercontent.com/109332566/189516599-81b5b905-f066-4c67-b397-4811923274b2.png">
<img width="358" alt="スクリーンショット 2022-09-11 16 05 29" src="https://user-images.githubusercontent.com/109332566/189516601-8f2f565e-334d-4d26-9de1-c3a28124ce10.png">